diff --git a/plr.cfg b/plr.cfg
index 18ba06e..4f33c19 100644
--- a/plr.cfg
+++ b/plr.cfg
@@ -47,6 +47,9 @@ timeout: 420.
 
 [gcode_macro RESUME_INTERRUPTED]
 gcode =
+    SET_KINEMATIC_POSITION X=0
+    SET_KINEMATIC_POSITION Y=0
+    SET_KINEMATIC_POSITION Z=0
     SET_GCODE_OFFSET Z=0 MOVE=0
     {% set z_height = params.Z_HEIGHT|default(printer.save_variables.variables.power_resume_z)|float %}
     {% set last_file = params.GCODE_FILE|default(printer.save_variables.variables.last_file)|string %}
