diff --git a/printer_eddy.cfg b/printer_eddy.cfg
index 0846d6e..1088532 100644
--- a/printer_eddy.cfg
+++ b/printer_eddy.cfg
@@ -29,4 +29,12 @@ y_offset: 21.42
 [temperature_probe btt_eddy]
 sensor_type: Generic 3950
 sensor_pin: eddy:gpio26
+min_temp: 0
+max_temp: 120
 horizontal_move_z: 2
+resting_z: 0.8
+calibration_position: 120, 120, 8
+calibration_extruder_temp: 160
+extruder_heating_z: 50
+calibration_bed_temp: 110
+max_validation_temp: 100
